node/resolvers/addTranslations.ts

```diff
@@ -0,0 +1,18 @@
+import addTranslations from '../resources/addTranslations'
+import { Entity, FailedTranslation, MessagesOfProvider } from '../typings/typings'
+
+interface AddTranslationsArgs {
+  entity: Entity
+  language: string
+  translations : MessagesOfProvider[]
+}
+
+async function addTranslationsResolver(
+  _: unknown,
+  { entity, language, translations }: AddTranslationsArgs,
+  ctx: Context
+): Promise<FailedTranslation[]> {
+  return await addTranslations(translations, entity, language, ctx)
```

Comment:
:pick: I don't think this needs to be an async function, since there's nothing to be done after the `await` statement. It could be replaced with
```ts
function addTranslationsResolver(
  _: unknown,
  { entity, language, translations }: AddTranslationsArgs,
  ctx: Context
): Promise<FailedTranslation[]> {
  return addTranslations(translations, entity, language, ctx)
}
```